tests/lsdb/catalog/test_catalog.py

```diff
@@ -345,10 +343,16 @@ def func(df, healpix):
     assert (small_sky_order1_catalog.skymap_histogram(func) == img).all()
 
 
-# pylint: disable=no-member
-def test_skymap_histogram_order_default(small_sky_order1_catalog, mocker):
-    mocker.patch("healpy.mollview")
+def test_skymap_histogram_empty(small_sky_order1_catalog):
+    def func(df, healpix):
+        return len(df) / hp.nside2pixarea(hp.order2nside(healpix.order), degrees=True)
+
+    expected_img = np.full(12, 1)
+    img = small_sky_order1_catalog.cone_search(0, 0, 1).skymap_histogram(func, default_value=1)
+    assert (img == expected_img).all()
+
 
+def test_skymap_histogram_order_default(small_sky_order1_catalog):
```

Comment:
It looks like you're missing the default value when initializing the `pixel_map` (line 362).

Reply:
Thanks, I changed the test to fix this